### DISCUSSION TITLE
11.0.10 dependency chain broken (Avalonia.Diagnostics)

### DISCUSSION THREAD

**Discussion by drei97** [AvaloniaUI/Avalonia#14839] [Q&A] [answered]
Hi,

I recently tried to update to 11.0.10 but got the following message

`Install failed (project: Ausprobieren, package: Avalonia.Diagnostics v11.0.10) Package restore failed. Rolling back package changes for 'Ausprobieren'. Detected package version outside of dependency constraint: Microsoft.CodeAnalysis.CSharp.Scripting 3.8.0 requires Microsoft.CodeAnalysis.Common (= 3.8.0) but version Microsoft.CodeAnalysis.Common 4.8.0-3.final was resolved. ... VisualStudio.Web.CodeGeneration.Core 8.0.1 -> Microsoft.VisualStudio.Web.CodeGeneration.Templating 8.0.1 -> Microsoft.CodeAnalysis.CSharp (>= 4.8.0-3.final) Ausprobieren -> Avalonia.Diagnostics 11.0.10 -> Microsoft.CodeAnalysis.CSharp.Scripting 3.8.0 -> Microsoft.CodeAnalysis.CSharp (= 3.8.0).`

Avalonia.Diagnostics is requiring Microsoft.CodeAnalysis packages by version 3.8.0. Installed is 4.8.0-3.final.

Since the CodeAnalysis packages are implicitly installed I can not change by hand.

Any hint to solve the issue on my side?

**Comment by timunie** [comment, marked as answer]
Make sure you don't have this package installed form somewhere else. EfCore for example ships it if you use `Microsoft.EntityFrameworkCore.Design`

**Reply by drei97** [reply]
Thanks for this hint.

The issue was with ASP.Net packages, remains from some testing. I removed them from my solution and the interference is gone.
